gcp/appengine/frontend3/src/templates/home.html

```diff
@@ -196,13 +197,47 @@ <h3 class="code-card-title">Scan directory recursively</h3>
             <mwc-icon-button class="icon" icon="content_copy"></mwc-icon-button>
           </clipboard-copy>
         </div>
+        <div class="code-card mdc-layout-grid__cell--span-6 mdc-layout-grid__cell--span-12-tablet">
+          <h3 class="code-card-title">Guided Remediation (basic)</h3>
+          <pre class="code-card-content" id="example-guided-remediation-basic">
+osv-scanner fix --non-interactive --strategy=in-place -L path/to/package-lock.json
+osv-scanner fix --non-interactive --strategy=relock -M path/to/package.json -L path/to/package-lock.json
+          </pre>
+          <clipboard-copy class="code-card-copy" for="example-guided-remediation-basic">
+            <mwc-icon-button class="icon" icon="content_copy"></mwc-icon-button>
+          </clipboard-copy>
+        </div>
+        <div class="code-card mdc-layout-grid__cell--span-6 mdc-layout-grid__cell--span-12-tablet">
+          <h3 class="code-card-title">Guided Remediation (advanced)</h3>
+          <pre class="code-card-content" id="example-guided-remediation-advanced">
+osv-scanner fix -M path/to/package.json -L path/to/package-lock.json
+          </pre>
+          <clipboard-copy class="code-card-copy" for="example-guided-remediation-advanced">
+            <mwc-icon-button class="icon" icon="content_copy"></mwc-icon-button>
+          </clipboard-copy>
+        </div>
       </div>
       <div class="mdc-layout-grid__cell--span-12">
         <div class="cta">
-          <a class="cta-primary link-button" href="https://github.com/google/osv-scanner">More details</a>
+          <a class="cta-primary link-button" href="https://google.github.io/osv-scanner/">More details</a>
         </div>
       </div>
     </div>
+    <div id="use-the-github-action" class="mdc-layout-grid__cell--span-12 github-action">
+      <h2 class="heading">GitHub Action (OSV&#8209;Scanner)</h2>
+      <p class="description">
+        The <a href="https://github.com/google/osv-scanner-action" target="_blank">OSV-Scanner GitHub Action</a>
```

Comment:
nit: we can make this a bit more concise.

OSV-Scanner also provides reusable GitHub workflows that can be easily integrated into CI/CD pipelines to provide continuous vulnerability scanning coverage. This can scan newly added dependencies in pull requests for introduced vulnerabilities, as well as perform regular vulnerability scans for the entire project.